Makefile

```
@@ -0,0 +1,23 @@
+CHART_NAME_1 ?= cray-vpa
```

Dont need the "_1" theres only one chart
```suggestion
CHART_NAME ?= cray-vpa
```